### PR TITLE
docs: Adjust community toolkit references

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/uno-community-toolkit.md [hunk 1]
```diff
@@ -143,31 +143,32 @@ Type universe cannot resolve assembly: Uno.UI, Version=255.255.255.255,
...
   <PackageReference Include="Uno.CommunityToolkit.WinUI.UI.Controls" Version="7.1.100" />
 </ItemGroup>
 ```
 
-Add a conditional reference:
+With a conditional reference:
 
 ```xml
-<ItemGroup Condition="'$(TargetFramework)' == 'net6.0-windows10.0.18362.0'">
+<ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">
```
Comment (Youssef1313) [inline doc/articles/uno-community-toolkit.md:160]: This will apply to the WPF build too, is it intended?

Comment (jeromelaban) [reply doc/articles/uno-community-toolkit.md:160]: In a class library, there's no `net7.0-windows` without the additional version, so yes, this is fine.

Comment (Youssef1313) [reply doc/articles/uno-community-toolkit.md:160]: Ah yes. Sorry, forgot that `net7.0-windows` is only in the Wpf-specific project but not in the classlib.

**Review by Youssef1313 — APPROVED** [review]
LGTM 🎉

**Pull request merged**